www/components/scroll/scroll.js

```diff
@@ -19,18 +19,21 @@ class scroll extends LitElement {
     // on hash change child event, scroll content
     const contains = (selector, text) => {
       const elements = this.querySelectorAll(selector);
-
-      return Array.prototype.filter.call(elements, (element) =>{
+      //convert NodeList to Array
+      const elementsArray = Array.from(elements);
+      return Array.prototype.filter.call(elementsArray, (element) =>{
```

Comment:
I think if we are going to use `.from`, then we also don't need to use `Array.prototype` since i think they were aiming to do the same thing?

So I think we could simply to this maybe?
```javascript
return Array.from(elements).filter(element) => {
  // do stuff
}
```